pkg/controller/stacks/install/installjob.go

```diff
@@ -58,7 +58,7 @@ type stackInstallJobCompleter struct {
 }
 
 func createInstallJob(i v1alpha1.StackInstaller, executorInfo *stacks.ExecutorInfo) *batchv1.Job {
-	ref := meta.AsOwner(meta.ReferenceTo(i, v1alpha1.StackGroupVersionKind))
+	ref := meta.AsOwner(meta.ReferenceTo(i, i.GroupVersionKind()))
```

Comment:
This fix makes sense.  do we have the same issue on these lines too?

* https://github.com/crossplaneio/crossplane/blob/master/pkg/controller/stacks/stack/stack.go#L229
* https://github.com/crossplaneio/crossplane/blob/master/pkg/controller/stacks/stack/stack.go#L264
* https://github.com/crossplaneio/crossplane/blob/master/pkg/controller/stacks/stack/stack.go#L299

Reply:
I think the other locations make sense.

When the Stack Manager creates the Deployment or Job or RBAC defined in the Stack fields it must set the owner of those resources to the Stack itself (`h.ext`) and use the GVK of a "Stack" -- `v1alpha1.StackGroupVersionKind`.

`createInstallJob` is run earlier in the processing of a Stack. The StackInstall manager intends to create the `Job` that will emits the Stack and CRD yaml resources.  This `Job` should reference the `StackInstall` or `ClusterStackInstall` resource that it is related to, and here the object and type are mismatched.

Nice find, @turkenh!

Reply:
Agree with @displague. Other lines look good to me.

Reply:
ah, got it, thank you for confirming those other instances @displague and @turkenh!